ddp-workspace/projects/ddp-dsm-ui/src/app/participant-page/participant-page.component.ts

```diff
@@ -1669,17 +1671,22 @@ export class ParticipantPageComponent implements OnInit, OnDestroy, AfterViewChe
       const countryQuestion = prequalActivity?.questionsAnswers?.find(questionAnswer => questionAnswer.stableId === this.SELF_COUNTRY);
       if (countryQuestion != null && countryQuestion.answer) {
         if (countryQuestion.answer instanceof Array) {
-          if (countryQuestion.answer.indexOf(this.SELF_COUNTRY_US) > -1) {
-            const stateQuestion = prequalActivity?.questionsAnswers?.find(questionAnswer => questionAnswer.stableId === this.SELF_STATE);
-            if (stateQuestion.answer instanceof Array) {
-              if (stateQuestion.answer.indexOf(this.SELF_STATE_NY) === -1) {
-                canBeSequencedBasedOnLocation = true;
+          const selfCountry = countryQuestion.answer[0];
```

Comment:
Can't think of multiple countries , hence using first element in the array should be safe